whipper/command/main.py

```diff
@@ -81,10 +77,10 @@ class Whipper(BaseCommand):
     subcommands = {
         'accurip': accurip.AccuRip,
         'cd':      cd.CD,
-        'debug':   debug.Debug,
         'drive':   drive.Drive,
         'offset':  offset.Offset,
-        'image':   image.Image
+        'image':   image.Image,
+        'mblookup': mblookup.MBLookup
```

Comment:
This lines doesn't align with the rest of the dict. They should probably all have another space added, or all made to only have one space total. (Of course, if you change `mblookup` to `lookup` then it should be fine without altering all the other lines.)

Reply:
Yeah I know-- I should un-align the rest of them for consistency.